extmod/modbluetooth.c

```diff
@@ -96,7 +96,7 @@ STATIC mp_obj_t bluetooth_uuid_make_new(const mp_obj_type_t *type, size_t n_args
         self->type = MP_BLUETOOTH_UUID_TYPE_16;
         mp_int_t value = mp_obj_get_int(all_args[0]);
         if (value > 65535) {
-            mp_raise_ValueError(MP_ERROR_TEXT("invalid UUID"));
+            mp_raise_ValueError(MP_ERROR_TEXT("invalid uuid"));
```

Comment:
What's the reason to make these UUID lowercase?

Reply:
Compression... I wanted to make them all the same case. `uuid` matches the argument name.

Reply:
But qstrs from the args would never be compressed along with error strings, right?